app/src/main/java/ar/com/wolox/android/example/ui/news/NewsFragment.kt

```diff
@@ -67,7 +66,16 @@ class NewsFragment @Inject constructor() : WolmoFragment<NewsPresenter>(), NewsV
         newsAdapter!!.addNews(news)
     }
 
+    override fun onNewClick(new: New) {
+        NewDetailActivity.start(requireContext(), new, context?.getSharedPreferences("ar.com.wolox.android.example.ui.login", Context.MODE_PRIVATE)!!.getInt("user_id", 0))
+    }
+
     private fun showToastNotification(messageId: Int) {
         Toast.makeText(activity, messageId, Toast.LENGTH_LONG).show()
     }
+
+    companion object {
+        fun newInstance() = NewsFragment()
+        private const val threshold = 3
```

Comment:
In general we use snake case for constants.
`        private const val THRESSHOLD = 3 `